tests/ops/op_math/test_exp.py

```diff
@@ -385,6 +384,11 @@ def test_sparse_matrix_wire_order_error(self):
 class TestDecomposition:
     """Test the decomposition of the `Exp` gate."""
 
+    # Order of `qml.ops.qubit.__all__` is not reliable, so
+    # must sort for consistent order in testing with multiple
+    # workers
+    all_qubit_operators = sorted(qml.ops.qubit.__all__)  # pylint: disable=no-member
```

Comment:
What exactly is the issue with having this unsorted?

Reply:
The sorting is related to the use of `pytest-xdist` to distribute tests across multiple CPU workers when testing on the server side. If the different worker processes receive marked parameters in different orders, a failure is raised in the test. This doesn't occur on a local machine unless the user installs `pytest-xdist` and runs the tests with the `-n 2` argument to ensure parallel testing (the server side testing does something similar automatically). By sorting, parametrized tests always run in the same order, avoiding this problem.